x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -72,6 +72,9 @@ export const WorkplaceSearchConfigured: React.FC<InitialAppData> = (props) => {
   const isOrganization = !pathname.match(personalSourceUrlRegex);
   setContext(isOrganization);
 
+  // We hide the chrome on app registration and need to unhide it for non-personal routes.
+  setChromeIsVisible(isOrganization);
```

Comment:
Can we a test for this in  `workplace_search/index.test.ts`? We can probably update the test on line 60 fairly quickly, like so:

```js
  it('renders layout, chrome, and header actions', () => {
    const wrapper = shallow(<WorkplaceSearchConfigured />);

    expect(wrapper.find(Layout).first().prop('readOnlyMode')).toBeFalsy();
    expect(wrapper.find(OverviewMVP)).toHaveLength(1);

    expect(mockKibanaValues.setChromeIsVisible).toHaveBeenCalledWith(true);
    expect(mockKibanaValues.renderHeaderActions).toHaveBeenCalledWith(WorkplaceSearchHeaderActions);
  });
```

Reply:
Another thought here - you might want to consider wrapping these actions in a `useEffect`, to reduce potential # of calls/rerenders and make the change scenarios clear. Example:

```js

  /**
   * Personal dashboard urls begin with /p/
   * EX: http://localhost:5601/app/enterprise_search/workplace_search/p/sources
   */
  useEffect(() => {
    const personalSourceUrlRegex = /^\/p\//g; // matches '/p/*'
    const isOrganization = !pathname.match(personalSourceUrlRegex); // TODO: Once auth is figured out, we need to have a check for the equivilent of `isAdmin`.

    setContext(isOrganization);
    setChromeIsVisible(isOrganization);
  }, [pathname]);
```